packages/perseus/src/components/sortable.tsx

```diff
@@ -432,15 +427,13 @@ class Sortable extends React.Component<SortableProps, SortableState> {
         margin: 5,
         onChange: function () {},
         linterContext: PerseusLinter.linterContextDefault,
-        waitForKatexLoad: true,
```

Comment:
@jeremywiebe we removed `waitForKatexLoad` during our pairing session, but I think removing it changes behavior, since it defaulted to `true`. Do we care about keeping this functionality? I have an idea for how to make it work:

- Render a hidden`TeX` component in `Sortable`
- Provide an `onRender` callback to the `TeX` that sets a `texReady` state variable on `Sortable`
- Render a spinner until `texReady` is true.

Reply:
I don't think we can guarantee that MathJax is loaded before the sortable renders. At least in the first incremental rollout I've planned for MathJax 3 (using it as the fallback instead of MathJax 2), MathJax will be behind a loadable, since it won't be needed on most pages.

I think we ought to keep this functionality and just change the name and implementation.